include/aspect/particle/interpolator/bilinear_least_squares.h

```diff
@@ -71,17 +71,11 @@ namespace aspect
         private:
           /**
            * Variables related to a limiting scheme that prevents overshoot and
-           * undershoot of interpolated particle properties based on global max
-           * and global min for each property.
+           * undershoot of interpolated particle properties based on the local
+           * max and min on the particles in that cell for each property.
            */
-          bool use_global_min_max_limiter;
 
-          /**
-           * For each interpolated particle property, a global max and global
-           * min are stored as elements of vectors.
-           */
-          std::vector<double> global_maximum_particle_properties;
-          std::vector<double> global_minimum_particle_properties;
+          ComponentMask use_linear_least_squares_limiter;
```

Comment:
Can you please add a comment for this variable? I assume the mask is set for each component where the limiter is enabled?